From 5609177532d5c69fe02d872c8f17ef8268e9a9a7 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Thu, 1 Jun 2023 16:47:15 +0200
Subject: [PATCH 0345/1141] arm64: defconfig: enable HASH support

Enable STM32 HASH support on 64 bits platform.

Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: I336d51666d660b27b283133415c682d76387d00e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/310343
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index de87bd450c42..9f3882277fe5 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -9,6 +9,7 @@ CONFIG_CAN_M_CAN_PLATFORM=m
 CONFIG_COUNTER=m
 CONFIG_CRYPTO_DEV_STM32_CRC=m
 CONFIG_CRYPTO_DEV_STM32_CRYP=m
+CONFIG_CRYPTO_DEV_STM32_HASH=m
 CONFIG_DRM=y
 CONFIG_DRM_BRIDGE=y
 CONFIG_DRM_DW_MIPI_DSI=y
-- 
2.39.2

